### DISCUSSION TITLE
[5.1.1] Why has "itemprop="inLanguage" been reverted in article default.php

### DISCUSSION THREAD

**Discussion by Scrabble96** [joomla/joomla-cms#43646] [Q&A]
### Problem identified
Having manually changed all my site's overrides earlier to remove all itemprops because of the use of JSON-LD, I now find I have to change them all back again for components/com_content/tmpl/article/default.php. (see PR #43123 ) It was explained to me some time ago that microdata and JSON-LD should not be mixed. But now it is. Why?

### Proposed solution
Find another way to solve the problem.

**Comment by brianteeman** [comment]
For the reasons stated in the linked pr
